data/presets/shop/toys.json

```diff
@@ -8,7 +8,8 @@
         "shop": "toys"
     },
     "terms": [
-        "games"
+        "games",
+        "teddy"
```

Comment:
I don't understand this alias.  Like a teddy bear?  Are we talking about places like buildabear.com ?

Reply:
![mr-bean-teddy](https://user-images.githubusercontent.com/4661658/171285171-7eb88493-3739-404b-8ca2-911a9dc3c76d.gif)